docs/tutorials/using_apis/mesh/thread_commissioning.md

```diff
@@ -55,6 +57,8 @@ Once you have completed the details, proceed to generate the QR code for your en
 
 #### Using the Thread commissioning application
 
+When building a Thread example application, do not change any of the PSKc related parameters (network name, extended PAN ID or PSKc). PSKc has been precalculated for the application. The nanostack has no capability to generate PSKc on runtime.
```

Comment:
no one has it is not limitation of nanostack but universe. only the one who knows the actual password can calculate those values

Reply:
So maybe better to remove this sentence -> The nanostack has no capability to generate PSKc on runtime. Should it be also here mentioned that we have used the "Thread Network" as a passwd?

Reply:
yes that is the most important information for the user. "

PSKc is calculated using network name and xpanid and "Thread Network" as network password that is required by commissioning application"

Something like that